newt/repo/repo.go

```diff
@@ -157,6 +173,32 @@ func NewRepoDependency(rname string, verstr string) (*RepoDependency, error) {
 	return rd, nil
 }
 
+func pickVersion(repo *Repo, versions []*Version) ([]*Version, error) {
+	fmt.Printf("Dependency list for %s contains a specific commit tag, " +
+		"so normal version number/stability comparison cannot be done.\n",
+		repo.Name())
+	fmt.Printf("If the following list does not contain the requirement to use, "+
+		"then modify your project.yml so that it does.\n")
+	for {
+		for i, vers := range versions {
+			fmt.Printf(" %d) %s\n", i, vers)
```

Comment:
I believe it calls the String() method for it automagically, presumably due to %s on the format string.
However, I can add the explicit call to make it more clear what's going on.